content/en/monitors/types/database_monitoring.md

```diff
@@ -8,13 +8,16 @@ kind: documentation
 
 With the [Database Monitoring (DBM)][1] monitor type, you can create monitors and alert on the data surfaced in DBM. These monitors can be configured to alert you when a DBM event type deviates from a predefined threshold over a given period of time.
 
-Some examples of these event types include when queries are waiting on the resolution of blocking queries or when active query executions exceed a given duration threshold.
+A few common monitoring scenarios (click for setup instructions):
```

Comment:
I've integrated this feedback in a slightly different way, since I want the primary emphasis to be on "these are just illustrative examples". But the result turned out to be less awkward than what I had here, thank you!